en/hosting_guide/configuration.rst

```diff
@@ -12,42 +230,56 @@ Cron jobs typically automate system maintenance.
 Cron jobs on Open Orchestra
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Open Orchestra has 3 cron jobs created with the provisioning:
+Open Orchestra has 4 cron jobs created with the provisioning:
+
+Site maps
+~~~~~~~~~
+
+Generate sitemap files for every sites, more information available in the `sitemap`_ documentation
+
+.. code-block::
+
+    0 2 * * * php /var/www/front-open-orchestra/current/app/console -e=prod orchestra:sitemaps:generate 2>> /tmp/cron.error.message
+
+Robots.txt
+~~~~~~~~~~
 
-* **Sitemap**: automatically generates sitemap files for every sites.
-* **Robots**: automatically generates the robots.txt files for every sites.
-* **ErrorPages**: automatically generates the special error pages files for every sites (eg 404 & 503 status).
+Generate the robots.txt files for every sites,
+further information about `robots`_
 
-Those tasks are launched every day at 2 am.
-The schedule of the tasks is define in the ansible cron role.
+.. code-block::
 
-For more information about the command which generates the sitemap.xml and the robots.txt see the documentation `sitemap and robots files`_
-For more information about the 404 and 503 special pages see the documentation `customizing error pages`_
+    0 2 * * * php /var/www/front-open-orchestra/current/app/console -e=prod orchestra:robots:generate 2>> /tmp/cron.error.message
 
-To modify the scheduling of the tasks, update variables in ``provisioning/roles/orchestraCron/vars/main.yml``.
+Error pages
+~~~~~~~~~~~
 
-Create a cron job
-~~~~~~~~~~~~~~~~~
+Generate the special error pages files for every sites (eg 404 & 503 status),
+for more information about the 404 and 503 special pages see the documentation `customizing error pages`_
 
-To add a cron job create a file named like your task in the ansible role ``provisioning/roles/orchestraCron/tasks``.
 
-In your file use the ansible cron command:
+.. code-block::
 
-.. code-block:: yaml
+    0 2 * * * php /var/www/front-open-orchestra/current/app/console -e=prod orchestra:errorpages:generate 2>> /tmp/cron.error.message
 
-    cron: name="task description" min="minutes" hour="hour" day="day" month="month" job="job"
+Error cron
+~~~~~~~~~~
 
-For more details about the cron command go see the documentation `ansible cron module`_.
+This cron sends an email if any of above cron didn't correctly.
 
-Add your task in the file ``provisioning/roles/orchestraCron/tasks/main.yml``
+.. code-block::
 
-.. code-block:: yaml
+    59 0-23 * * * if [ -s '/tmp/cron.error.message' ]; then  cat /tmp/cron.error.message | mailx -s "cron error" contact@open-orchestra.com; fi; rm /tmp/cron.error.message;
 
-    - include: task.yml
+Ansible
+-------
 
-The available variables are in ``provisioning/roles/orchestraCron/vars/main.yml``.
+If you don't want set the different configurations (Virtual Host, Varnish, Cron ) manually, you can use
```

Comment:
There is a problem here in preview mode: the first part of this sentence is in bold